test/minnum.ll

```diff
@@ -0,0 +1,39 @@
+; RUN: llvm-as %s -o %t.bc
```

Comment:
For the test naming please take a look at https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/780
And for the test location please mind https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/780#issuecomment-708219326

Reply:
There's a bunch of other intrinsic-related tests that should be moved likewise. Would it be fine if I renamed/moved the new test, and factored out all other related updates into a separate PR?

Reply:
@AGindinson, I haven't yet decided about particular test location for LLVM intrinsics, so, it is okay to me to leave the test in its current location. However, I do agree that it would be better if you add `llvm.` prefix to its name

Reply:
Fine, I've moved `llvm.minnum.ll` back to the original directory, and am ready to follow-up with a renaming PR for other files.